src/helpers/errorHandlers.ts

```diff
@@ -137,3 +137,16 @@ export function checkEmptyResult(responses: Execution.IExecutionResponses) {
 
     return responses;
 }
+
+export const validateLocalIdentifiers = (buckets: VisualizationObject.IBucket[]) => {
```

Comment:
How about using uniq from lodash:

```
const localIdentifiers = buckets.map(bucket => bucket.localIdentifier);
const uniqueEntries = uniq(localIdentifiers);

return uniqueEntries.length === localIdentifiers.length;
```

Reply:
Even better way, without map:

```
uniqBy(buckets, bucket => bucket.localIdentifier)
```

Reply:
maybe even 

```
uniqBy(buckets, 'localIdentifier');
```

is enough https://lodash.com/docs/4.17.10#uniqBy

Reply:
so oneliner :D 

```
const hasDuplicateIdentifiers = buckets => (buckets.length !== uniqBy(buckets, 'localIdentifier').length);
```

Reply:
solved differently